README.md

```diff
@@ -203,7 +203,7 @@ easy to fork and contribute any changes back upstream.
    Please make sure `eval "$(pyenv init -)"` is placed toward the end of the shell
    configuration file since it manipulates `PATH` during the initialization.
     ```sh
-    $ echo 'eval "$(pyenv init -)"' >> ~/.bash_profile
+    $ echo '[ -x $PYENV_ROOT/bin/pyenv ] && eval "$(pyenv init -)"' >> ~/.bash_profile
```

Comment:
As for Bash and other modern shells, using `[[` instead of `[` is better.
And then also use it in the `eval` - otherwise it might still fail if `$PYENV_ROOT` is not in the `$PATH`.